pkg/common/config/config.go

```diff
@@ -90,6 +90,9 @@ const (
 	// DefaultListVolumeThreshold specifies the default maximum number of differences in volumes between CNS
 	// and kubernetes
 	DefaultListVolumeThreshold = 50
+	// SupervisorIDPrefix is added before the SupervisorID
+	// Using this CNS UI can form an appropriate URL to navigate from CNS UI to WCP UI
+	SupervisorIDPrefix = "vSphereSupervisorID-"
```

Comment:
As we are using this constant in the same package/file, can we rename it to `supervisorIDPrefix` instead of `SupervisorIDPrefix`?

Reply:
yes let me do that